test/models/deploy_group_test.rb

```diff
@@ -1,18 +1,30 @@
 require_relative '../test_helper'
 
 describe DeployGroup do
-  let(:prod_env) { environments(:production) }
+  let(:stage) { stages(:test_staging) }
+  let(:environment) { environments(:production) }
+  let(:deploy_group) { deploy_groups(:pod1) }
+
+  def self.it_expires_stage(method)
+    it "expires stages when #{method}" do
+      stage.deploy_groups << deploy_group
+      stage.update_column(:updated_at, 1.minute.ago)
```

Comment:
create is even more expensive then an update_column :)

On Mon, Jun 22, 2015 at 3:11 PM, Roman Sandler <notifications@github.com>
wrote:

> In test/models/deploy_group_test.rb
> <https://github.com/zendesk/samson/pull/479#discussion_r32990009>:
>
> > @@ -1,18 +1,30 @@
> >  require_relative '../test_helper'
> >
> >  describe DeployGroup do
> > -  let(:prod_env) { environments(:production) }
> > +  let(:stage) { stages(:test_staging) }
> > +  let(:environment) { environments(:production) }
> > +  let(:deploy_group) { deploy_groups(:pod1) }
> > +
> > +  def self.it_expires_stage(method)
> > +    it "expires stages when #{method}" do
> > +      stage.deploy_groups << deploy_group
> > +      stage.update_column(:updated_at, 1.minute.ago)
>
> My suggestion was to make an old fixture - not sure what we can do here
> but in factorygirl you could go let(:stage) { create(:stage, updated_at:
> 1.minute.ago) } or similar
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/479/files#r32990009>.
>